From 86245ed9e16ec390bb316e382b8859e2b6168a01 Mon Sep 17 00:00:00 2001
From: fli <fupan.li@windriver.com>
Date: Tue, 12 Nov 2013 14:10:44 +0800
Subject: [PATCH 14/14] arch: add the PINCTRL to the X86 Arch

Intel also supports the PINCTRL on their SoC, such as BayTrail.

Signed-off-by: fli <fupan.li@windriver.com>
---
 arch/x86/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index eaefbcf..2b57a3a 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -121,6 +121,7 @@ config X86
 	select OLD_SIGACTION if X86_32
 	select COMPAT_OLD_SIGACTION if IA32_EMULATION
 	select RTC_LIB
+	select PINCTRL
 
 config INSTRUCTION_DECODER
 	def_bool y
-- 
1.8.4.93.g57e4c17

